auth-service/src/test/kotlin/uk/gov/communities/delta/controllers/DeltaLoginControllerTest.kt

```diff
@@ -222,8 +222,8 @@ class DeltaLoginControllerTest {
         every { failedLoginCounter.increment(1.0) } returns Unit
         every { successfulLoginCounter.increment(1.0) } returns Unit
         coEvery { userAuditService.userFormLoginAudit(any(), any()) } returns Unit
-        coEvery { authorizationCodeService.generateAndStore(any(), any(), any()) } answers {
-            AuthCode("test-auth-code", "user", client, Instant.now(), "trace")
+        coEvery { authorizationCodeService.generateAndStore(any(), any(), any(), any()) } answers {
```

Comment:
Ideally, we'd have a verify somewhere that generateAndStore was called with isSso is false. But looks like it's currently not verified at all...

So I'd say, (if it's easy), could you add a suitable verify to maybe the testLoginPostSuccess test?